docs/build_node.md

```diff
@@ -208,17 +221,19 @@ Filter tx_filter = Filter.new_impl(new Filter.FilterInterface() {
     }
 
     @Override
-    Option_C2Tuple_usizeTransactionZZ register_output(WatchedOutput output) {
+    public Option_C2Tuple_usizeTransactionZZ register_output(WatchedOutput output)
+    {
         // <insert code for you to watch for any transactions that spend this
-        // output on-chain>
-        // See blockdata.md for more info. XXX HOW2LINK???
```

Comment:
This should ideally link to blockdata.md, I'm not sure how to insert a link.

Reply:
I linked it below in References + Implementation Notes